src/stdlib/stdlib/libs/deploy.tact

```diff
@@ -2,7 +2,7 @@
 ///
 /// See: https://docs.tact-lang.org/ref/stdlib-deploy#deploy
 ///
-message Deploy {
+message(2490013878) Deploy {
```

Comment:
let's maybe use hexadecimal numbers here; it can be mapped to 32-bit opcodes easily